Week1/homework/app.js

```diff
@@ -2,10 +2,104 @@
 {
   const bookTitles = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'dont_tell_me_your_afraid', 'dont_tell_mummy', 'positive_thinking', 'time_management', 'The alchemist', 'the_power_of_now', 'ekadoli', 'Before_we_were_yours', 'hebrew_female_in_my_heart', 'amareeta'
   ];
 
-
   // Replace with your own code
-  console.log(bookTitles);
+  //  2.3
+  function makeBooksTitlesList() {
+    const ul = document.createElement('ul');
+    ul.innerHTML = 'Books Titles: ';
+    document.getElementsByTagName('body')[0].appendChild(ul);
+    for (let i = 0; i < bookTitles.length; ++i) {
+      const li = document.createElement('li');
+      li.innerHTML = bookTitles[i];
```

Comment:
The content of the `bookTitles` array that you created is not intended for dispIay. The purose of these titles so that  _you can use these later as HTML id attributes_. This is also mentioned as a hint at the end of step 2.7.

So in this loop where your create and and append `li` elements you should not set the `innerHTML` but do this:

```js
li.setAttribute('id', bookTitles[i]);
```